doc/UsersGuide/BuildingRunningTesting/RunSRW.rst

```diff
@@ -629,75 +617,344 @@ The output files (in ``.png`` format) will be located in the ``postprd`` directo
+snowfall (``'ASNOW'``) and its associated obs type (:term:`NOHRSC`).  In other words, all the (meta)tasks
+in :numref:`Table %s <VXWorkflowTasksTable>`.  will be included in the Rocoto XML except for those
+associated with the :term:`NOHRSC` obs type and the ``'ASNOW'`` field group.  Users might want to set
+``VX_FIELD_GROUPS`` in this way for example because the forecast experiment they are verifying is for a
+summer period for which ``ASNOW`` is not relevant.
+
+
+Staging Observation Files
+``````````````````````````````````
+The taskgroup in ``verify_pre.yaml`` defines a set of workflow tasks named ``get_obs_*`` --- where the ``*``
+represents any one of the supported obs types :term:`CCPA`, :term:`NOHRSC`, :term:`MRMS`, and :term:`NDAS` ---
+that will first check for the existence of the obs files required for VX at the locations on disk specified
+by the variables ``*_OBS_DIR`` and ``OBS_*_FN_TEMPLATES[1,3,...]`` in the ``verification:`` section of
+``config.yaml``.  The ``*_OBS_DIR``  are the base directories in which the obs files are or should be
+staged, and the ``OBS_*_FN_TEMPLATES[1,3,...]`` are the file name templates (with METplus time strings
+used for templating; see example below).  The ``[1,3,...]`` in ``OBS_*_FN_TEMPLATES[1,3,...]`` refer
+to the second, fourth, etc elements of ``OBS_*_FN_TEMPLATES`` and correspond to the various sets of files
+that the obs type contains.  (The first, third, etc elements, i.e. ``OBS_*_FN_TEMPLATES[0,2,...]``,
+indicate the VX field groups for which the respective sets of obs files are used to verify and should
+not be changed.) Note that ``OBS_*_FN_TEMPLATES[1,3,...]`` may include leading subdirectories and are
+relative to the obs type's ``*_OBS_DIR``.  Thus, the templates for the full paths to the obs files are
+given by
+
+.. code-block:: console
+
+   {*_OBS_DIR}/{OBS_*_FN_TEMPLATES[1,3,...]}
+
+If the obs files exist at the locations specified by these variables, then the ``get_obs_*`` tasks will
+succeed and the workflow will move on to subsequent tasks.  If one or more obs files do not exist, the
+``get_obs_*`` tasks will attempt to retrieve the required files from a data store such as NOAA HPSS and
+place them in the locations specified by ``*_OBS_DIR`` and ``OBS_*_FN_TEMPLATES[1,3,...]``.  Assuming
+that attempt is successful, the workflow will move on to subsequent tasks.  Thus:
+
+   * Users who have the obs files already available (staged) on their system only need to set ``*_OBS_DIR``
+     and ``OBS_*_FN_TEMPLATES[1,3,...]`` in ``config.yaml`` to match those staging locations and file names.  
+   
+   * Users who do not have the obs files available on their systems and do not have access to NOAA HPSS
+     need to download :term:`CCPA`, :term:`NOHRSC`, :term:`MRMS`, and/or :term:`NDAS` files manually
+     from collections of publicly available data such as the ones listed `here <https://dtcenter.org/nwp-containers-online-tutorial/publicly-available-data-sets>`__. 
```

Comment:
@gsketefian This link doesn't work anymore :-/

Reply:
@gspetro-NOAA I thought you had put in that link!  I did a lot of merging and conflict resolving because I was working with multiple branches on multiple computers, so I may have picked up this link from an old version of the app a few PRs back.  I'm happy to change this to whatever you think is best (removing it is fine by me).

Reply:
Lol I originally had, but DTC eventually retired that page, so I removed the link when the link checker flagged it. I'll remove here, too, because I wasn't able to find a good substitute link. Feel free to recommend one tho if you know of any!

Reply:
Ok, I reworded a bit. See if that works! I put back "are," although I think that "should be" covers both meanings and is more concise, but I don't feel the need to die arguing on that hill. ;)